examples/src/examples/misc/multi-app/example.mjs

```diff
@@ -21,9 +21,7 @@ const createApp = async function (deviceType) {
         device = new pc.WebgpuGraphicsDevice(canvas, {});
         await device.initWebGpu(rootPath + '/static/lib/glslang/glslang.js', rootPath + '/static/lib/twgsl/twgsl.js');
     } else if (deviceType === 'webgl1' || deviceType === 'webgl2') {
-        device = new pc.WebglGraphicsDevice(canvas, {
```

Comment:
Do we want to add a warning if `webgl1` is requested saying deprecated or something?

Reply:
No really, we'll string it out from examples completely so it cannot be requested by the examples